api/src/validation.php

```diff
@@ -37,6 +37,9 @@
             'firstname' => v::optional(v::notEmpty()->noWhitespace()->length(1, 51)),
             'password' => v::notEmpty()->length(1, 256),
             'email' => v::optional(v::notEmpty()->length(1, 256)::email())
+        ],
+        'getUser' => [
```

Comment:
Remove this validation as the validation is already done by the expression matching of the route:
`/users/{id:[0-9]+`